### DISCUSSION TITLE
Can RangeSlider only have one changeable parameter?

### DISCUSSION THREAD

**Discussion by Njitram2000** [algolia/instantsearch#6144] [Q&A]
I want a slider for stock that selects the minimum amount of stock. So the slider should represent 1 value.

I can use https://www.algolia.com/doc/api-reference/widgets/range-slider/js/ for this but this one always has 2 values: the min and the max. If I hide the tab to slide the min, I can still click on the bar and it changes the min value. It want the min to be locked at 0 so that THE value of the slider is max.

**Comment by Haroenv** [comment]
maybe you want to put the max on the actual maximum of stock you have, but min being the one you manipulate?

**Reply by Njitram2000** [reply]
If you do not specify max or min, the lowest and highest value of the facet in the search results is used. But you still have the 2 tabs on either side to change the values of the range for the filtering. So that does not solve my issue.
I want 1 tab, like in the standard html range input: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/range

**Reply by Haroenv** [reply]
You can use a connectRange in that case, use the regular `input type range` and only set the `min` value, leaving the `max` to the whole range.

**Reply by Njitram2000** [reply]
That is indeed what I have been doing in the meantime.
